toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -188,6 +188,10 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 	logger.Log.Infof("Number of failed SRPMs:            %d", len(failures))
 	logger.Log.Infof("Number of blocked SRPMs:           %d", len(unbuiltSRPMs))
 	logger.Log.Infof("Number of unresolved dependencies: %d", len(unresolvedDependencies))
+
+	logger.Log.Infof("Number of RPMs conflicts: : %d", len(rpmConflicts))
+	logger.Log.Infof("Rebuild toolchain flag: %s", supressConflictingRPMs)
```

Comment:
I'd remove that line here, since it's not really part of the build summary. It's closer to the build parameters/configuration. Instead, I'd suggest making the `(ignored if ALLOW_TOOLCHAIN_REBUILDS=y)` message below conditional and then we could just simply say `(ignored because ALLOW_TOOLCHAIN_REBUILDS=y)`.

```suggestion
```